src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp

```diff
@@ -98,55 +102,42 @@ auto partial_derivatives(
                                   tmpl::size_t<Dim>, DerivativeFrame>>;
 
 namespace Tags {
-namespace Tags_detail {
-template <typename Tag, typename VolumeDim, typename Frame>
-struct deriv_impl<Tag, VolumeDim, Frame,
-                  Requires<tt::is_a_v<Tensor, db::item_type<Tag>>>>
-    : db::PrefixTag, db::SimpleTag {
-  using type = TensorMetafunctions::prepend_spatial_index<
-      db::item_type<Tag>, VolumeDim::value, UpLo::Lo, Frame>;
-  using tag = Tag;
-  static std::string name() noexcept { return "deriv"; }
-};
 
-// Partial derivatives with derivative index in the frame related to the logical
-// coordinates by InverseJacobianTag
-template <typename... VariablesTags, typename... DerivativeTags,
-          typename InverseJacobianTag>
-struct deriv_impl<
-    tmpl::list<VariablesTags...>, tmpl::list<DerivativeTags...>,
-    InverseJacobianTag,
-    Requires<tt::is_a_v<Tensor, db::item_type<InverseJacobianTag>>>>
-    : db::ComputeTag {
+/*!
+ * \ingroup DataBoxTagsGroup
+ * \brief Compute the spatial derivatives of tags in a Variables
+ *
+ * Computes the spatial derivatives of the Tensors in the Variables represented
+ * by `VariablesTag` in the frame mapped to by `InverseJacobianTag`. You can
+ * optionally specify a subset of the tags in the Variables as `GradientTags` to
+ * only take the gradient of those.
+ *
+ * This tag inherits from `db::variables_tag_with_tags_list<VariablesTag,
```

Comment:
Why do you need to document what it inherits from? Is it necessary to know this when I use the tag? Wouldn't it be more beneficial to document that you can use `deriv<Tag>` or whatever to grab the individual differentiated tensors from the databox?

Reply:
The base class is, in this case, an important part of the interface, because it tells you what tag you can retrieve the result of the compute item as (and for some reason doxygen does not show public base classes).  It implies that you can retrieve the tensors using `deriv<Tag>` because it shows that this is a variables tag and all variables tags work that way.

Reply:
The purpose should be made clear in the documentation, and why not document the other tags and how to retrieve them?

Reply:
Perhaps "may be retrieved via" or similar instead of "inherits from"?

I feel that we can assume some knowledge of how to use the DataBox when documenting DataBox tags.  We don't document how to retrieve tensors for all the other tags that obtain subitems through (possibly) containing Variables (`dt`, `deriv`, `ComputeDiv`, `NormalDotNumericalFlux`, ...), so we should come to some consistent decision on whether we want to put this everywhere.  At the moment common practice is to not do so.

Reply:
Yea, sorry for not being clear. I think using `May be retrieved via` would be perfect.

Good point. I think it would be sufficient to say what the appropriate base tag to use would be. So in this case saying something like `The variables may be retrieved via deriv<...>, while the tensors of the Variables may be retrieved using deriv<...>` (expanding on the ...) would be a good way to proceed and also should apply in general for any prefix and base tag combination. That should make it possible to easily figure out what the best way to retrieve an item is. What do you think?